pkg/storage/couchdb/couchdbstore_test.go

```diff
@@ -0,0 +1,226 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package couchdbstore
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/trustbloc/edge-core/pkg/storage"
+
+	"github.com/go-kivik/kivik"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	couchDBURL    = "localhost:5984"
+	testStoreName = "teststore"
+	testKey       = "sampleDBKey"
+	testValue     = `{"JSONKey":"JSONValue"}`
```

Comment:
you assume the value always JSON

Reply:
@troyronda  @bstasyszyn @llorllale @rolsonquadras

Reply:
https://github.com/hyperledger/fabric/blob/21fc6f619d9a1d6bc83280ec6d53bc22b2e96f92/core/ledger/util/couchdb/couchdb.go#L632

Reply:
Per our discussion on Friday with @bstasyszyn , I updated the CouchDB store Put method so that a non-JSON payload will be saved as an attachment. I verified that JSON payloads do get merged with the CouchDB document (this should answer @bstasyszyn's question from Friday)